source/IntegrationConfigFactory.ts

```diff
@@ -591,13 +591,403 @@ export default class IntegrationConfigFactory {
+                        + 'FROM PS_TRNS_TEST_DTL'
+                });
+                PEOPLESOFT_TEMPLATE_STATEMENTS.push({
+                    name: 'PS_TST_CREDIT_COMP',
+                    query: 'Select * from PS_TST_CREDIT_COMP'
+                });
+                PEOPLESOFT_TEMPLATE_STATEMENTS.push({
+                    name: 'PSXLATITEM',
+                    query: 'Select * from PSXLATITEM'
+                });
+                PEOPLESOFT_TEMPLATE_STATEMENTS.push({
+                    name: 'PS_UA_DT_SEQ',
+                    query: 'Select UA_DT_UNITS, UA_DT_REQ_SEM, UA_DT_REQ_PRIORITY, REQUIREMENT, EFFDT, RQ_LINE_KEY_NBR '
+                        + 'FROM PS_UA_DT_SEQ'
+                });
+                PEOPLESOFT_TEMPLATE_STATEMENTS.push({
+                    name: 'PS_UA_DT_STD_RSLT',
+                    query: 'Select EMPLID, UA_DT_PLNR_CRS_SEQ, ITEM_R_STATUS, UA_DT_SOURCE, STRM, UA_DT_COMP_STRM, CRSE_ID, UA_DT_COMP_CRSE_ID, '
+                        + 'CRSE_OFFER_NBR,	UA_DT_UNITS, UA_DT_REQ_SEM, UA_DT_REMOVED, UA_DT_MODE '
+                        + 'FROM PS_UA_DT_STD_RSLT'
+                });
+                PEOPLESOFT_TEMPLATE_STATEMENTS.push({
+                    name: 'PS_SAA_ADB_Results',
+                    query: "Select EMPLID, ANALYSIS_DB_SEQ,SAA_ENTRY_SEQ, RPT_DATE, TSCRPT_TYPE, ENTRY_R_TYPE, RQ_DATE, RQRMNT_GROUP, "
+                        + "RQ_GRP_LINE_NBR, REQUIREMENT, RQ_LINE_NBR, UNITS_REQUIRED, SAA_UNITS_USED, CRSES_REQUIRED, SAA_CRSES_USED, "
+                        + "GPA_ACTUAL, ITEM_R_STATUS, SAA_PRINT_CNTL, SAA_CAREER_RPT "
+                        + "FROM PS_SAA_ADB_Results WHERE RPT_DATE >= To_Date('01/01/2017 01:01:01 AM', 'MM/DD/YYYY HH:MI:SS AM')"
+                });
+                PEOPLESOFT_TEMPLATE_STATEMENTS.push({
+                    name: 'PS_SAA_ADB_COURSES',
+                    query: "SELECT DISTINCT EMPLID, ANALYSIS_DB_SEQ, CRSE_TAG, RPT_DATE, TSCRPT_TYPE, Earn_Credit, grading_basis_enrl, "
+                        + "UNT_EARNED, INCLUDE_IN_GPA, GRADE_POINTS, CRSE_ID, CATALOG_NBR, SAA_CAREER_RPT, RQMNT_DESIGNTN, SUBJECT "
+                        + "FROM PS_SAA_ADB_COURSES WHERE RPT_DATE >= To_Date('01/01/2017 01:01:01 AM', 'MM/DD/YYYY HH:MI:SS AM')"
+                });
+                PEOPLESOFT_TEMPLATE_STATEMENTS.push({
+                    name: 'PS_SAA_ADB_CRSEUSE',
+                    query: "SELECT RQRMNT_GROUP, REQUIREMENT, RQ_LINE_NBR, SAA_ENTRY_SEQ, SAA_COURSE_SEQ, Emplid, analysis_db_seq, crse_tag, "
+                        + "TSCRPT_TYPE, SAA_CAREER_RPT, RPT_DATE, SEL_PROCESS_TYPE, SEL_MODE, UNT_EARNED "
+                        + "FROM PS_SAA_ADB_CRSEUSE WHERE RPT_DATE >= To_Date('01/01/2017 01:01:01 AM', 'MM/DD/YYYY HH:MI:SS AM')"
+                });
+                PEOPLESOFT_TEMPLATE_STATEMENTS.push({
+                    name: 'ps_saa_adb_crseavl',
```

Comment:
I'm not sure why some table names are upper case and others are lower cases. @croutledgeaais Does this cause any issues during the step that gathers the DDL for tables that are lower-cased?

Reply:
I was wondering about that too - I kept casing how I found it in the spokes. I'd like to normalize them if I knew for a fact what kind of effect it would have (or hopefully NOT have) downstream.

Reply:
@croutledgeaais Sounds good. I'll get that fixed, along with your other suggestions, at lunch today.